src/scripts/main.js

```diff
@@ -1,3 +1,294 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('table');
+const header = document.querySelector('thead');
+const categories = header.querySelector('tr');
+const tbody = [...table.tBodies][0];
+const rows = tbody.rows;
+
+// sorting
+categories.addEventListener('click', (e) => {
+  const sorted = e.target.classList.contains('ascending')
+    ? sortDesc(e.target, rows, e.target.cellIndex)
+    : sortAsc(e.target, rows, e.target.cellIndex);
+
+  tbody.append(...sorted);
+});
+
+function sortAsc(target, collection, i) {
+  const title = target.textContent;
+  const array = [...collection];
+
+  switch (true) {
+    case (title === 'Name' || title === 'Position' || title === 'Office') :
+
+      array.sort((a, b) => {
+        const aWord = [...a.cells][i].textContent;
+        const bWord = [...b.cells][i].textContent;
+
+        return aWord.localeCompare(bWord);
+      });
+
+      break;
+
+    case (title === 'Age') :
+      array.sort((a, b) => {
+        return [...a.cells][i].textContent - [...b.cells][i].textContent;
+      });
+
+      break;
+
+    case (title === 'Salary') :
+      array.sort((a, b) => {
+        const aConverted = xChange([...a.cells][i].textContent);
+        const bConverted = xChange([...b.cells][i].textContent);
+
+        return aConverted - bConverted;
+      });
+
+      break;
+  }
+
+  [...categories.cells].forEach(cell => cell.classList.remove('ascending'));
+
+  target.classList.add('ascending');
```

Comment:
instead of adding and removing classes to DOM elements, you can create a variable setAscending, which will store a boolean value, which you can set to opposite on every click and sort table according whether it is true or false